pkg/client/rpc_client.go

```diff
@@ -263,6 +265,19 @@ func (r *RPCClient) logResult(
 			append(results, "err", err)...,
 		)
 	}
+
+	metrics.RPCCallLatency.
+		WithLabelValues(
+			metrics.EVM,                    // chain family
+			r.chainID.String(),             // chain id
+			rpcDomain,                      // rpc url
+			"false",                        // is send only
+			strconv.FormatBool(err == nil), // is successful
+			callName,                       // rpc call name
+		).
+		Observe(float64(callDuration))
+
+	// TODO: Remove deprecated metric
```

Comment:
just curious, why marking as deprecated and leaving a TODO? some compatibility issue?

Reply:
We need to keep the old metrics to not break any dashboards people may have for backwards compatibility, but eventually everyone should move to the chain-agnostic ones.